### PR TITLE
[hdpowerview] Add additional hub properties

### LINKED ISSUE #13173: [hdpowerview] Add additional Hub properties from user data
According to the official API documentation, the `GET /api/userdata` request response exposes some information which could be useful as hub thing properties. For example:
- Firmware versions (also available via `GET /api/fwversion`)
- Serial number
- MAC address
- IP address/gateway/netmask
- Hub name
- Timezone/location/offset
- Brightness/color of LED
- Auto-backup enabled
- and more...

### PR DESCRIPTION
**Pull request by jlaur**
Resolves #13173 by adding the following hub properties:
- Serial number
- MAC address
- Hub name

In the future serial number or MAC address can potentially replace host as representation property.

Example:
image

### CODE REVIEW

**Review by lolodomo — APPROVED** [review]
LGTM

File: bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/handler/HDPowerViewHubHandler.java [hunk 1]
```diff
@@ -98,7 +99,7 @@ public class HDPowerViewHubHandler extends BaseBridgeHandler {
     private List<Scene> sceneCache = new CopyOnWriteArrayList<>();
     private List<SceneCollection> sceneCollectionCache = new CopyOnWriteArrayList<>();
     private List<ScheduledEvent> scheduledEventCache = new CopyOnWriteArrayList<>();
-    private Instant firmwareVersionsUpdated = Instant.MIN;
+    private Instant UserDataUpdated = Instant.MIN;
```
Comment (andrewfg) [inline bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/handler/HDPowerViewHubHandler.java:102]: Field name should be camelCase

**Review by andrewfg — COMMENTED** [review]
@jlaur just a minor issue concerning a field name

File: bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/handler/HDPowerViewHubHandler.java (hunk 1 above)
Comment (jlaur) [reply bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/handler/HDPowerViewHubHandler.java:102]: Whoops! Thanks for noticing. However, PR was already merged, so will fix this and other SAT findings after #13175 has been merged.

Comment (jlaur) [reply bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/handler/HDPowerViewHubHandler.java:102]: Actually, besides a few findings in tests, this was the only SAT finding, so renamed this single variable within that PR.

**Pull request merged**